src/nvim/marktree.c

```diff
@@ -1131,6 +1266,7 @@ void marktree_clear(MarkTree *b)
   }
   map_destroy(uint64_t, b->id2node);
   b->n_keys = 0;
+  memset(b->meta_root, 0, kMTMetaCount * sizeof(b->meta_root[0]));
```

Comment:
(nit/asking if this is preferred because that's what I used in https://github.com/neovim/neovim/pull/27128/files#diff-6886911a605fae58e382d1e7aadde7d9ea8d5de47121c408eab4f9ab8df92d5eR313)
```suggestion
  CLEAR_FIELD(b->meta_root));
```

Reply:
The point here is just to be consistent with other usages of other meta counts which are not fields (maybe it works in theory also there by accident but I prefer being explicit with nested arrays instead of macro magic).